### PR TITLE
Glossary update

### PR DESCRIPTION
**Pull request by jesseyuen**
This PR includes terms submitted by the newest BAT family members, the Register team.

### CODE REVIEW

File: app/glossary.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ A collection of terms commonly used across the Becoming a Teacher service lines.
 Accreditation
 : All initial teacher training providers must be ‘accredited’ by the Secretary of State for Education. This process of ‘accreditation’ confirms they have permission to offer courses leading to Qualified Teacher Status (QTS). (NB: Ofsted is the body which checks the courses providers offer meet the standards set by the department.)
 : The exception is the School Direct (SD) training route; SDs need a ‘ratifying accredited provider’. The ratifying accredited provider may be a School Centred Initial Teacher Training (SCITT) or a Higher Education Institute (HEI), and is responsible for maintaining the quality of the training provided.
-: As accredited providers, SCITTs can offer QTS courses but do not have ‘degree awarding powers’. So to cater to candidates seeking academic awards (for example, a Post Graduate Certificate in Education (PGCE)), they need to partner with an HEI.
+: As accredited providers, SCITTs can offer QTS courses but do not have ‘degree awarding powers’. So to cater to candidates seeking academic awards (for example, a Post Graduate Certificate in Education (PGCE), they need to partner with an HEI.
```
Comment (paulrobertlloyd) [inline app/glossary.md]: That second closing bracket is needed - read the sentence again! Maybe this can be reworded to avoid nested bracketing?

**Review by paulrobertlloyd — COMMENTED** [review]
Just the one niggle.

File: app/glossary.md (hunk 1 above)
Comment (adamsilver) [inline app/glossary.md]:
```suggestion
: As accredited bodies, SCITTs can offer QTS courses but do not have the power to award a degree. To enable candidates who seek an academic award like a PGCE (Post Graduate Certificate in Education) they need to partner with an HEI.
```

File: app/glossary.md [hunk 2]
```diff
@@ -106,9 +115,18 @@ Teacher Analysis Division (TAD)
 Teaching School Alliance
 : Outstanding schools that work with others to provide high quality training to new and experienced staff. Cannot be lead schools or providers.
 
+Teacher Regulation Agency (TRA)
```
Comment (paulrobertlloyd) [inline app/glossary.md]: Ordering should be alphabetical:
* Teacher Analysis Division (TAD)
* Teacher Qualifications Unit (TQU)
* Teacher Reference Number (TRN)
* Teacher Regulation Agency (TRA)
* Teacher supply model (TSM)
* Teaching School Alliance

Comment (jesseyuen) [reply app/glossary.md]: 🤦‍♀️

**Review by paulrobertlloyd — APPROVED** [review]
:shipit:

**Pull request merged**